About this release IS 6.0.0 (#3222)
wso2/docs-is: en/docs/references/about-this-release.md

WSO2 Identity Server 6.0.0 is the latest WSO2 Identity Server (WSO2 IS) release and is the successor of WSO2 Identity Server 5.11.0.

## WHat's new in this release?

WSO2 IS 6.0.0 contains the following new features and enhancements:

-   **Integration with TypingDNA**

    TypingDNA is a behavioral biometrics vendor and a pioneer in delivering typing biometrics technology as an API for user-friendly authentication to businesses across cybersecurity, finance, education, and retail. The vendor leverages typing biometrics to provide customers with a seamless, user-friendly, risk-based authentication (RBA) experience to enhance security and fraud detection.

    Learn more

-   **Integration with ELK for identity analytics** 

    Easy integration with ELK to view login and session-related analytics data. 

    [Learn more]({{base_path}}/guides/elk-analytics/elk-analytics/)

-   **Multi-attribute login support** 

    This simplifies user sign in by allowing users to sign in with their preferred attributes such as email, phone number, or username as the login identifier.

    [Learn more]({{base_path}}/guides/login/multi-attribute-login/)

-   **Device flow support** 

>     This expands the range of devices an organization can use within their overall solution by including devices with limited user input capabilities (such as smart TVs which do not have keyboards). With device flow support, users can leverage other devices, such as smartphones, to complete login on the limited input device.

Reviewer: ```suggestion
    This expands the range of devices an organization can use within their overall solution by including devices with limited user input capabilities (such as smart TVs which do not have keyboards). With device flow support, users can leverage other devices, such as smartphones, to complete login on a device with limited input.
```

    [Learn more]({{base_path}}/guides/access-delegation/try-device-flow/)

-   **PBKDF2 hashing for user passwords**

>     This is a simple cryptographic key derivation function, which is resistant to brute force attacks. PBKDF2 is recommended by NIST and PBKDF2 is required to achieve FIPS-140 compliance.

Reviewer: ```suggestion
    This is a simple cryptographic key derivation function, which is resistant to brute force attacks. PBKDF2, recommended by NIST, is required to achieve FIPS-140 compliance.
```

    [Learn more]({{base_path}}/deploy/environment-compatibility/)

-   **Authentication SDKs**

    SDKs allow you to integrate web or single-page applications easily with WSO2 Identity Server and OpenID Connect, while adhering to security best practices. The following SDKs are supported:

    - [React SDK]({{base_path}}/sdks/start-integrating-apps/integrate-a-react-app/)
    - [Angular SDK]({{base_path}}/sdks/start-integrating-apps/integrate-an-angular-app/)
    - [JavaScript SDK]({{base_path}}/sdks/start-integrating-apps/integrate-a-js-app/)

-   **Passwordless authentication with Magic Link**

    Magic link is a form of passwordless authentication. It allows users to log in by clicking a link sent to their email instead of entering a password.

    [Learn more]({{base_path}}/guides/passwordless/magic-link/)

-   **FIDO attestation**

    FIDO attestation validations allow you to further validate the FIDO2 authenticator data during the security key registration.

    Learn more

-   **Federated IDP Initiated OIDC Back-Channel Logout**

>     When it comes to OIDC identity federation in Identity Server, WSO2 IS acts as an RP to the federated identity provider. However, currently there is no mechanism to terminate the sessions and revoke tokens in WSO2 IS (RP), whenever there is a session update in the federated IDP (OP) side. 

Reviewer: ```suggestion
    When it comes to OIDC identity federation in Identity Server, WSO2 IS acts as a Relying Party (RP) to the federated identity provider. However, currently there is no mechanism to terminate the sessions and revoke tokens in WSO2 IS (RP), whenever there is a session update in the federated IDP (OP) side. 
```
